### DISCUSSION TITLE
Google Summer of Code 2024 CNCF Project Proposals

### DISCUSSION THREAD

**Discussion by aliok** [cncf/mentoring#1116] [Announcements]
Hello!

CNCF has been participating in the Google Summer of Code during the last few years and now we are ready to start the 2024 program cycle.
 
All CNCF projects are eligible to apply to participate in the Google Summer of Code. Please start working on the project ideas, and submit them via pull request: https://github.com/cncf/mentoring/blob/main/programs/summerofcode/2024.md#ideas. 

The deadline to submit project ideas is March 11, 2024 (though we'd be happy for project ideas to come in sooner than that if possible to increase our chance of getting accepted as an org).

[Google Summer of Code 2024 Timeline](https://developers.google.com/open-source/gsoc/timeline)

Thanks!

**Comment by vr-varad** [comment]
how about adding a chatbot inside the cncf glossary page? just strikes my mind

**Comment by vr-varad** [comment]
@aliok what u think about it

**Reply by aliok** [reply]
@vr-varad I would recommend talking about that to the CFCF Glossary team.

**Comment by aliok** [comment]
🎉  CNCF is accepted to Google Summer of Code 2024 : https://github.com/cncf/mentoring/discussions/1186

**Comment by aliok** [comment]
Closing this discussion as we now have a new one for the new phase (https://github.com/cncf/mentoring/discussions/1186)
